### PR TITLE
#159304051 Add endpoint to delete a person

### PR DESCRIPTION
**Pull request by Dheavyman**
#### What does this PR do?
- Have an endpoint to delete a person in the application
#### Description of Task to be completed?
- An authenticated user should be able to delete a person and get a successful message
#### How should this be manually tested?
- Create a local copy with the following commands: `git fetch origin ft-delete-a-person-159304051:ft-delete-a-person-159304051`, then `git checkout ft-delete-a-person-159304051`
- Start the application with the command: `python manage.py runserver`
- Using postman, send a DELETE request to: `localhost:5000/api/v1/centers/:center_id/people/:person:id`
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#159304051](https://www.pivotaltracker.com/story/show/159304051)
#### Postman collection:
- [Link](https://www.getpostman.com/collections/2d8c1aa4aad75d3cddb0)
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def create_app(config=config[config_name]):
     db.init_app(app)
 
     # import all models
-    from api.models import User, Asset, AssetCategory, Attribute, Center
+    from api.models import User, Asset, AssetCategory, Attribute, Center, User
```
Comment (GbengaOshinaga) [inline main.py]: @Dheavyman You imported `User` twice

Comment (Dheavyman) [reply main.py]: Thanks for this, fixed.

File: api/utilities/validators/validate_id.py [hunk 2]
```diff
@@ -17,10 +17,13 @@ def validate_id(func):
 
     @wraps(func)
     def decorated_function(*args, **kwargs):
-        if not is_valid_id(kwargs.get('id', None)):
-            raise ValidationError({
-                'status': 'error',
-                'message': serialization_errors['invalid_id']
-            }, 400)
+        for key in kwargs:
```
Comment (ibutiti) [inline api/utilities/validators/validate_id.py]: Nice

File: tests/test_person_resource/test_delete_person_endpoint.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
+"""Module to test endpoint to delete a person"""
```
Comment (ibutiti) [inline tests/test_person_resource/test_delete_person_endpoint.py]: Please add tests for:
1. Center already deleted
2. User already deleted

Comment (Dheavyman) [reply tests/test_person_resource/test_delete_person_endpoint.py]: @ibutiti I have a test for the case of a center that is not existing or has been deleted and that of a person already. Check line 75 below and if there is more feedback or clarification on that let me know

File: api/utilities/validators/validate_id.py (hunk 2 above)
Comment (Dheavyman) [reply api/utilities/validators/validate_id.py]: Thanks

File: tests/test_person_resource/test_delete_person_endpoint.py (hunk 3 above)
Comment (Dheavyman) [reply tests/test_person_resource/test_delete_person_endpoint.py]: @ibutiti In such case the resource is essentially not existing. The non-existing is a name to signify that the center or person id has been deleted or is a valid id but not existing entirely on the db.

Comment (Dheavyman) [reply tests/test_person_resource/test_delete_person_endpoint.py]: @ibutiti All cases tested and ready for approval. Thanks

**Comment by Dheavyman** [comment]
@frankip Yeah, you are right. But that will be done in the create a person/user task. I can't do that here because there are changes that will be made to user model which I don't have yet.

**Pull request merged**